client/src/test/resources/AssociativeArrayOfLists.yaml

```diff
@@ -1,6 +0,0 @@
-# Block associated array of lists (showing examples of both inline and blocked lists)
```

Comment:
@sebady Is this an inadvertent removal? I see test skips in the new dataprovider module as a result of this file (and the other 6 below) removed as test resources